frontend/src/components/ActivityItem.tsx

```diff
@@ -2,23 +2,23 @@ import React, { FC } from 'react';
 
 type ActivityItemProps = {
   user: string;
-  project: string;
   message: string;
 };
 
 const ActivityItem: FC<ActivityItemProps> = ({
   user,
-  project,
   message,
 }: ActivityItemProps) => {
   return (
-    <div className="relative">
-      <div className="bg-white m-6 py-6">
-        <h1 className="font-medium text-xl">{user}</h1>
-        <h3>{project}</h3>
-        <p>{message}</p>
+    <>
+      <div className="relative flex text-left items-center pl-3">
+        <div className="w-12 h-12 bg-accent rounded-full cursor-pointer"></div>
+        <div className="bg-white m-6 py-3">
+          <h1>{user}</h1>
+          <p className="font-medium text-xl">{message}</p>
+        </div>
       </div>
-    </div>
+    </>
```

Comment:
You don't need <> and </> here
They're only needed if you are returning multiple children elements. In this case, `<div className="relative flex text-left items-center pl-3">` is the only child element